src/qt/walletmodel.cpp

```diff
@@ -506,12 +504,15 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
         questionString.append(tr("Warning: This may pay the additional fee by reducing change outputs or adding inputs, when necessary. It may add a new change output if one does not already exist. These changes may potentially leak privacy."));
     }
 
-    SendConfirmationDialog confirmationDialog(tr("Confirm fee bump"), questionString);
+    // Show the Craete Unsigned button if the option is set or private keys are disabled
+    bool show_unsigned_button = getOptionsModel()->getEnablePSBTControls() || m_wallet->privateKeysDisabled();
+
+    SendConfirmationDialog confirmationDialog(tr("Confirm fee bump"), questionString, "", "", SEND_CONFIRM_DELAY, "Send", !m_wallet->privateKeysDisabled(), "Create Unsigned", show_unsigned_button, nullptr);
```

Comment:
shouldn't `"Send"` and `"Create Unsigned"` be translatable? 

```suggestion
    SendConfirmationDialog confirmationDialog(tr("Confirm fee bump"), questionString, "", "", SEND_CONFIRM_DELAY, tr("Send"), !m_wallet->privateKeysDisabled(), tr("Create Unsigned"), show_unsigned_button, nullptr);
```

Reply:
Done